tests/arithmetic/ast_arithmetic.re

```diff
@@ -0,0 +1,9 @@
+/* Auto-generated by codegen_type */
+type assignment_statement = (variable, string, expression, string)
```

Comment:
It would be nice to sort those type definitions by dependencies, with the entry type at the top (e.g., program), and
the leaves at the bottom (e.g., number) or inversely, but to have an order that helps to follow the flow.

Reply:
ok addressed. See the generated files now